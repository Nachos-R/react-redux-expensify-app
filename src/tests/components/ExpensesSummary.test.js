import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render ExpensesSummary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={234} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpensesSummary with multiply expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={42} expensesTotal={23454656789097} />);
    expect(wrapper).toMatchSnapshot();
});