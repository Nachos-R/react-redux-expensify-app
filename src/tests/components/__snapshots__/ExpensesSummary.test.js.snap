// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.34
  </h1>
</div>
`;

exports[`should correctly render ExpensesSummary with multiply expense 1`] = `
<div>
  <h1>
    Viewing 
    42
     
    expenses
     totalling 
    $234,546,567,890.97
  </h1>
</div>
`;
